docs/api/associations/index.md

```diff
@@ -1,11 +1,12 @@
 <a name="associations"></a>
```

Comment:
Please don't update in docs/api - its autogenerated.

All your changes should be reflected in the code, otherwise they will disappear next time the docs are generated